connector.rb

```diff
@@ -126,8 +122,6 @@
             end
           when 'demo'
             'https://oauth.4me-demo.com/token'
-          else
```

Comment:
This will break recipes with existing custom domain connections. Is it expected behaviour?

Reply:
It was clarified here, https://github.com/Workato-Partners/4me/pull/13#discussion_r1519548016

Reply:
@szaretsky, @thawalk, when will connector be available? Thanks